clang/test/CIR/CodeGen/complex.cpp

```diff
@@ -176,3 +176,27 @@ void foo7() {
 // OGCG: store float %[[TMP_A]], ptr %[[C_REAL_PTR]], align 4
 // OGCG: store float 2.000000e+00, ptr %[[C_IMAG_PTR]], align 4
 
+void foo9(double r, double i) {
```

Comment:
```suggestion
void foo9(double a, double b) {
```
For consistency with what you've called these variables in the checks.